File: LDCoreData-OC/PeopleEntity+CoreDataProperties.m
//
//  PeopleEntity+CoreDataProperties.m
//  LDCoreData-OC
//
//  Created by LiuQiqiang on 2018/4/21.
//  Copyright © 2018年 LiuQiqiang. All rights reserved.
//
//

#import "PeopleEntity+CoreDataProperties.h"

@implementation PeopleEntity (CoreDataProperties)

+ (NSFetchRequest<PeopleEntity *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"PeopleEntity"];
}

@dynamic age;
@dynamic name;
@dynamic sex;

@end

File: LDCoreDataTool-Swift/PeopleEntity+CoreDataProperties.swift
//
//  PeopleEntity+CoreDataProperties.swift
//  LDCoreDataTool2-Swift
//
//  Created by Artron_LQQ on 2017/7/3.
//  Copyright © 2017年 Artup. All rights reserved.
//

import Foundation
import CoreData


extension PeopleEntity {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<PeopleEntity> {
        return NSFetchRequest<PeopleEntity>(entityName: "PeopleEntity")
    }

    @NSManaged public var age: Int16
    @NSManaged public var name: String?
    @NSManaged public var sex: Bool

}
